cmd/server/assets/codes/bulk-issue.html

```diff
@@ -0,0 +1,302 @@
+    integrity="sha512-nOQuvD9nKirvxDdvQ9OMqe2dgapbPB7vYAMrzJihw5m+aNcf0dX53m6YxM4LgA9u8e9eg9QX+/+mPu8kCNpV2A=="
+    crossorigin="anonymous"></script>
+</head>
+
+<body id="users-import" class="tab-content">
+  {{template "navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <div class="card mb-3 shadow-sm">
+      <div class="card-header">Bulk issue</div>
+      <div class="card-body">
+        {{template "beta-notice" .}}
+
+        {{if not $hasSMSConfig}}
+        <div class="alert alert-danger">
+          <span class="oi oi-warning"></span>
+          No SMS provider is configured for this realm.
+          Please contact a realm administrator to enable this feature.
+        </div>
+        {{end}}
+
+        <p>
+          Use this form to bulk issue codes.
+        </p>
+
+        <pre>Example file contents:
+          <code>
+            +12065555555,2020-11-10
+          </code>
+        </pre>
+
+        <p>
+          This form will data and issue a code for each provided entry.
+          Phone numbers are expected to be in <a href="https://www.twilio.com/docs/glossary/what-e164" target="_blank">E.164</a> format.
+        </p>
+        <form id="form">
+          <div class="form-label-group input-group">
+            <input type="text" class="form-control" id="retry-code" placeholder="Retry code" required>
+            <label for="retry-code">Retry code</label>
+            <div class="input-group-append">
+              <button class="btn btn-outline-secondary" id="new-code" type="button">New code</button>
+            </div>
+          </div>
+          <small class="form-text text-muted pb-2">
+            The retry code is a password for this bulk upload status in case it fails during processing.
+            If you attempt this upload again with the same retry code, the server will be
+            able to retry re-issue codes for phones which have not been successfully notified.
+            You may use the same retry code for all of your uploads, but guard it like a password.
+          </small>
+          <div class="form-check">
+            <input type="checkbox" class="form-check-input" id="remember-code">
+            <label class="form-check-label text-secondary" for="remember-code">Remember code</label>
+          </div>
+          <small class="form-text text-muted pb-4">
+            Your browser will remember this retry code.
+          </small>
+
+          <div class="form-label-group input-group">
+            <input type="text" class="form-control" id="start-at" placeholder="Start at line" value="0">
+            <label for="start-at">Start at line</label>
+          </div>
+          <small class="form-text text-muted pb-4">
+            Parsing the CSV will start at this line. Begin at 0 for a new file.
+            This can be used to resume a canceled or partial upload.
+          </small>
+
+          <div class="custom-file">
```

Comment:
This order feels backward. I think I should select the file, then pick options, then start the upload.